main.py

```diff
@@ -1,10 +1,44 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    # Skill.objects.all().delete()
+    # Player.objects.all().delete()
+    # Guild.objects.all().delete()
+    # Race.objects.all().delete()
+    with open("players.json") as f:
+        players = json.load(f)
+
+    for nickname, fields in players.items():
+        race_info = fields["race"]
+        guild_info = fields["guild"]
+
+        race, _ = Race.objects.get_or_create(
+            name=race_info["name"],
+            description=race_info["description"])
+
+        for skill in race_info["skills"]:
+            Skill.objects.get_or_create(
+                name=skill["name"],
+                bonus=skill["bonus"],
+                race=race,
+            )
+        if guild_info:
+            guild, _ = Guild.objects.get_or_create(
+                name=guild_info["name"],
+                description=guild_info["description"])
+        else:
+            guild = None
```

Comment:
Ypu can create this viriable at the start and do not use else block .

If your condition is triggered, the contents of the variable will simply be overwritten